modules/LinearBoltzmannSolvers/LBSMIPSteadyState/lbsmip_01_initialize.cc

```diff
@@ -0,0 +1,108 @@
+#include "lbsmip_steady_solver.h"
+
+#include "LBSSteadyState/Acceleration/diffusion_mip.h"
+
+/**Initializing.*/
+void lbs::MIPSteadyStateSolver::Initialize()
+{
+  PerformInputChecks();                //a
+
+  options_.scattering_order = 0; //overwrite any option set for this
+
+  PrintSimHeader();                    //b
+
+  MPI_Barrier(MPI_COMM_WORLD);
+
+  InitMaterials();                     //c
+  InitializeSpatialDiscretization();   //d
+
+  InitializeGroupsets();           //e
+  gs_mip_solvers_.assign(groupsets_.size(), nullptr);
+
+//  ComputeNumberOfMoments(); is not called           //f
+  num_moments_ = 1;
+  InitializeParrays();                       //g
+  InitializeBoundaries();                    //h
+  InitializePointSources();                  //i
+
+  // Initialize source func
+  using namespace std::placeholders;
+  active_set_source_function_ =
+    std::bind(&SteadyStateSolver::SetSource, this, _1, _2, _3);
+
+  //============================================= Initialize groupset solvers
+  const size_t num_groupsets = groupsets_.size();
+  for (size_t gs=0; gs<num_groupsets; ++gs)
+  {
+    const auto& groupset = groupsets_[gs];
+
+    //=========================================== Make UnknownManager
+    const size_t gs_G = groupset.groups.size();
+    chi_math::UnknownManager uk_man;
+    uk_man.AddUnknown(chi_math::UnknownType::VECTOR_N, gs_G);
+
+    //=========================================== Make boundary conditions
+    typedef chi_mesh::sweep_management::BoundaryType SwpBndryType;
+    typedef lbs::acceleration::BoundaryCondition BC;
+    typedef lbs::acceleration::BCType BCType;
+
+    std::map<uint64_t, BC> bcs;
+    for (auto& [bid, lbs_bndry] : sweep_boundaries_)
+    {
+      if (lbs_bndry->Type() == SwpBndryType::REFLECTING)
+        bcs[bid] = {BCType::ROBIN,{0.0,1.0,0.0}};
+      else if (lbs_bndry->Type() == SwpBndryType::VACUUM)
+        bcs[bid] = {BCType::ROBIN,{0.25,0.5}};
+    }//for sweep-boundary
+
+    //=========================================== Make xs map
+    typedef lbs::acceleration::Multigroup_D_and_sigR MGXs;
+    typedef std::map<int, lbs::acceleration::Multigroup_D_and_sigR> MapMatID2XS;
+    MapMatID2XS map_mat_id_2_mgxs;
```

Comment:
To be consistent with other naming conventions, this should be `matid_to_mgxs_map`.